frontend/src/components/applications/tabs/SchoolPoliciesTab.vue

```diff
@@ -681,7 +681,7 @@
         >If 'no' to any of the above policies, please explain below:</v-label
       >
       <v-row>
-        <!--TODO: Additional logic to force requirement if any bool is false? -->
+        &lt;!&ndash;TODO: Additional logic to force requirement if any bool is false? &ndash;&gt;
```

Comment:
not sure if you intended to have this here. looks odd compared to the rest of the code.